### ISSUE TITLE
decodeEscapeSequence() does not support non latin char

### ISSUE THREAD

**Issue by 7dir** [sebhildebrandt/systeminformation#415]
`utils.js` does not support non latin char

```js
function decodeEscapeSequence(str, base) {
  base = base || 16;
  return str.replace(/\\x([0-9A-Fa-f]{2})/g, function () {
    return String.fromCharCode(parseInt(arguments[1], base));
  });
}
```

similar
https://github.com/sebhildebrandt/systeminformation/issues/317

```
\xd0\x97\xd0\xb0\xd1\x80\xd0\xb5\xd0\xb7\xd0\xb5\xd1\x80\xd0\xb2\xd0\xb8\xd1\x80\xd0\xbe\xd0\xb2\xd0\xb0\xd0\xbd\xd0\xbe
```
should return
```
Зарезервировано
```

**Comment by 7dir** [comment]
you can test your test text here https://2cyr.com/decode/?lang=ru

**Comment by 7dir** [comment]
`lsblk -P` return hex-escaped

```
       -P, --pairs
              Produce output in the form of key="value" pairs.  All potentially unsafe characters
              are hex-escaped (\x<code>).
```

**Comment by 7dir** [comment]
we should replace
```js
line = decodeURIComponent(line.replace(/\\x/g, '%'))
```

**Comment by sebhildebrandt** [comment]
Fixed, version 4.27.6 released
